hoomd/pytest/test_local_snapshot.py

```diff
@@ -84,13 +84,13 @@
                   value=[5, 2, 3, 2, 5],
                   new_value=[2, 1, 0.5, 1, 2],
                   shape=(Np,)),
-    image=dict(np_type=np.signedinteger,
+    image=dict(np_type=np.int32,
```

Comment:
Is this raising warnnings on CI? I don't get any warnings locally which is strange.

Reply:
I get the warnings locally on an Apple M1 MacOs system with numpy 1.20.1. I do not get them on the Linux systems I develop on. I do not know why.

The warning is:
```
lib/python3.9/site-packages/numpy/core/function_base.py:175: DeprecationWarning: Converting `np.integer` or `np.signedinteger` to a dtype is deprecated. The current result is `np.dtype(np.int_)` which is not strictly correct. Note that the result depends on the system. To ensure stable results use may want to use `np.int64` or `np.int32`.
    return y.astype(dtype, copy=False)
```